types/common.ts

```diff
@@ -2,7 +2,27 @@
  * TODO: implement the type
  * https://github.com/ConnecMent/dydx-bot/issues/1
  */
-export type Candle = any;
 
 export type Pair = string;
-export type PositionType = "long" | "short" | null;
+export type PositionType = 'long' | 'short' | null;
+export type TimeFrame = string;
+export interface Candle {
+  startedAt: string;
+  ticker: string;
+  resolution:
+    | '1MIN'
+    | '5MINS'
+    | '15MINS'
+    | '30MINS'
+    | '1HOUR'
+    | '4HOURS'
+    | '1DAY';
+  low: string | number;
```

Comment:
You stated that these are all `string`s in the API response. So the `| number` union item is not required.

Reply:
yes all of them are string . okay I remove them .